diplomarbeit/app.vue

```diff
@@ -1,61 +1,78 @@
       </h1>
-      <img src="assets/img/schriftlogo.svg" class="mx-12 my-2 sm:h-24 sm:mt-3 sm:mb-28"
+      <img src="assets/img/schriftlogo.svg" class="mx-12 my-2 lg:h-24 lg:mt-3 lg:mb-28 max-w-80 lg:max-w-none"
            alt="Schriftlogo des Projekts ScrumpliCity, es zeigt den Schriftzug ScrumpliCity mit einer Illustration von 3 Häusern welche von einem Kreis, welcher an Scrum erinnert, umkreist sind">
-      <p class="sm:scroll-mt-36 text-sc-orange mt-12 sm:mt-16 text-2xl sm:text-4xl mb-6 sm:mb-7 font-heading"
+      <p class="lg:scroll-mt-36 text-sc-orange mt-12 lg:mt-16 text-2xl lg:text-4xl mb-6 lg:mb-7 font-heading"
          id="projektbeschreibung">
         Unser Projekt
       </p>
-      <p class="text-justify sm:mb-20 text-sm sm:text-xl leading-6 sm:leading-9 mx-8 sm:mx-[400px] max-w-[80ch] hyphens-manual">
+      <p class="text-justify text-sm lg:text-xl leading-6 lg:leading-9 mx-8 lg:mx-[400px] max-w-[80ch] hyphens-manual">
         ScrumpliCity entsteht im Rahmen einer Diplomarbeit an der HTL Rennweg. Wir entwickeln eine
         Webanwendung, welche spielerisch die
-        Projektmanagement&shy;methode Scrum vermitteln soll. Lernende bauen in Kleingruppen Städte aus Papier und erlangen
+        Projektmanagement&shy;methode Scrum vermitteln soll. Lernende bauen in Kleingruppen Städte aus Papier und
+        erlangen
         währenddessen Einblicke in den Ablauf eines Scrum-Projekts. Unsere Web-Applikation leitet parallel dazu die
         verschiedenen Phasen von Scrum. Unterstützt werden die Lernenden auch durch vorgefertigte Bastelvorlagen.
       </p>
-      <p class="sm:scroll-mt-36 mt-28 sm:mt-0 text-sc-orange text-2xl sm:text-4xl mb-6 sm:mb-7 font-heading"
-         id="ziele">
-        Unsere Ziele
+      <p class="lg:scroll-mt-36 mt-28 lg:mt-20 text-sc-orange text-2xl lg:text-4xl mb-6 lg:mb-7 font-heading"
+         id="features">
+        Features unserer Website
       </p>
-      <ul class="list-disc list-inside text-sm sm:text-xl leading-7 sm:leading-10 mx-8">
+      <ul class="list-disc list-inside text-sm lg:text-xl leading-7 lg:leading-10 ml-4">
         <li>Lehrkraftprofile inkl. Registrierung & Login</li>
         <li>Einfache Raumübersicht für die Lehrkraft</li>
-        <li>Gesamter Spielablauf
-          <ul class="list-disc list-inside ml-4">
-            <li>Backlog mit User Storys</li>
+        <li>Erstes Scrum-Projekt spielerisch vermitteln und begleiten
+          <ul class="list-inside ml-8" style="list-style: circle">
+            <li>Online zu verfassende Sprint-Backlogs</li>
             <li>Sprint-Plannings</li>
-            <li>Bauphase mit Timer</li>
-            <li>Review-Meeting inkl. Velocity</li>
+            <li>Bauphase mit Timer
+              <ul class="list-inside ml-8" style="list-style: square">
+                <li>Physische Stadt aus Papiervorlagen wird von Schülern gebaut</li>
+              </ul>
+            </li>
+            <li>Review-Meetings inkl. Velocity Berechnung</li>
```

Comment:
```suggestion
            <li>Review-Meetings inkl. Velocity-Berechnung</li>
```
da gehört im Deutschen ein Bindestrich